config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -90,7 +90,8 @@ presubmits:
 
   - name: pull-kubernetes-e2e-kind-ipv6
     cluster: k8s-infra-prow-build
-    optional: false
+    # TODO: once this is no longer required in github, disable always_run
+    optional: true
```

Comment:
why don't disable always_run directly? making it optional will not change much

Reply:
because of the branch protector, we automatically configure github which statuses are `Required`

that happens async. we don't want it to be marked required when we're not going to run it.

phasing in/out required jobs has always been painful, I think this two step approach will be smoother

Reply:
It's running smoother these days: https://prow.k8s.io/?job=ci-test-infra-branchprotector

But sometimes it's been quite slow. Currently it takes about 15m and runs every hour. So latency in the worst case could be ~1h unless we manually start a new protector run sooner.